src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
 import './App.scss';
+import React, { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const queryLowerCase = query.toLowerCase().trim();
+
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => (
+    title.toLowerCase().includes(queryLowerCase)
+      || description.toLowerCase().includes(queryLowerCase)
+  ));
```

Comment:
No need to filter in case when query is empty
```suggestion
  const visibleMovies = !!query 
    ? moviesFromServer.filter(({ title, description }) => (
    title.toLowerCase().includes(queryLowerCase)
      || description.toLowerCase().includes(queryLowerCase)
    : moviesFromServer
  ));
```